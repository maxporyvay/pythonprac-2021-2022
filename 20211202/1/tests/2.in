class G(metaclass=dump):
    def __init__(self, val, val2=10):
        self.val = val
        self.val2 = val2
    def __str__(self):
        return str(self.val + self.val2)
		
g = G(4)
w = G(2, 6)
print(g)
print(w)