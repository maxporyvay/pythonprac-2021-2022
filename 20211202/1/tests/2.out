__init__: (4,), {}
__init__: (2, 6), {}
__str__: (), {}
14
__str__: (), {}
8
